ui/js/app.js

```diff
@@ -1662,4 +1700,32 @@ function shouldEnableTouch() {
   return chromiumPlus73 || chromePlus73;
 }
 
+function setAvailableCoupons(areThereAvailableCoupons) {
+  if (areThereAvailableCoupons) {
+    $('#insert-first-bill-coupon-added').hide();
+    $('#choose-fiat-coupon-added').hide();
+    $('#insert-first-bill-promo-button').show();
+    $('#choose-fiat-promo-button').show();
+  } else {
+    $('#insert-first-bill-promo-button').hide();
+    $('#choose-fiat-promo-button').hide();
+  }
+}
+
+function setCurrentDiscount(currentDiscount) {
+  if (currentDiscount > 0) {
+    $('#insert-first-bill-promo-button').hide();
+    $('#choose-fiat-promo-button').hide();
+    $('#insert-first-bill-coupon-added').html('\u2714 Coupon added (' + currentDiscount + '% discount)');
+    $('#choose-fiat-coupon-added').html('\u2714 Coupon added (' + currentDiscount + '% discount)');
+    $('#insert-first-bill-coupon-added').show();
+    $('#choose-fiat-coupon-added').show();
+  } else {
+    $('#insert-first-bill-promo-button').show();
+    $('#choose-fiat-promo-button').show();
+    $('#insert-first-bill-coupon-added').hide();
+    $('#choose-fiat-coupon-added').hide();
+  }
+}
+
 //# sourceMappingURL=app.js.map
```

Comment:
remove annotation and add newline at the end of the file

Reply:
This is an auto-generated file, so I won't touch it